rpcserver/router.go

```diff
@@ -87,6 +87,16 @@ func (server *routedBoltzServer) GetInfo(_ context.Context, _ *boltzrpc.GetInfoR
 		pendingReverseSwapIds = append(pendingReverseSwapIds, pendingReverseSwap.Id)
 	}
 
+	refundableSwaps, err := server.database.QueryRefundableSwaps()
```

Comment:
Thats given implicitly since any swap will attempt to be refunded cooperatively immediately after it errored. If that doesnt work it will show up in this query until the script path where it will still need a manual refund if no address / wallet is avilable.